src/vec/vec2.rs

```diff
@@ -75,7 +75,17 @@ macro_rules! vec2s {
             }
 
             /// The wedge (aka exterior) product of two vectors.
-            ///
+
```

Comment:
need /// at this line, i.e. the skipped line still needs the doc comment marker thingy